.github/CODEOWNERS

```diff
@@ -1,17 +1,101 @@
-*                          @senior7515 @dotnwat
```

Comment:
> 🥁  the time has come to nix @senior7515 as a default maintainer ☠️☠️☠️☠️ 

LGTM

Reply:
haha I think there is now no default so don't feel bad :P